src/routes/+layout.svelte

```diff
@@ -4,18 +4,8 @@
   import "@fontsource/poppins"
 
   import "$styles/_init.scss"
-  import "svelte-highlight/styles/github.css"
-
-  // Highlight.js (svelte-highlight) code block styling
-  import githubDark from "svelte-highlight/styles/github-dark"
-  import "svelte-highlight/styles/github-dark.css"
```

Comment:
I think this caused the bug with styles in Code blocks. Multiple imports + applied in two different ways, not sure if it was supposed to work. Code blocks in guides still highlighted with tokyo styles, code blocks in schemas are different and styled differently too, nothing has changed,